mender-auth/cli/keystore.cpp

```diff
@@ -70,6 +70,7 @@ const error::Error KeyStoreStaticKeyError = error::Error(
 error::Error MenderKeyStore::Load() {
 	auto exp_key = crypto::PrivateKey::LoadFromPEM(key_name_, passphrase_);
 	if (!exp_key) {
+		log::Debug("Error loading private key from " + key_name_ + ": " + exp_key.error().message);
 		return KeyStoreNoKeysError;
```

Comment:
This is exactly the scenario where the error should be constructed in place, using the extra context that you have in the log message above. Then you might not even need the log message, because the caller will print it.